chatops-settings.yaml

```diff
@@ -0,0 +1,140 @@
+
```

Comment:
Similar to https://github.com/StackStorm/st2-dockerfiles/pull/19 `st2chatops.env` is part of the `st2chatops` deb/rpm package and there is no need to include a copy of these vars here.
Especially considering that it could become quickly out of date.